crates/api_models/src/enums.rs

```diff
@@ -427,7 +427,8 @@ pub enum PaymentMethod {
     Card,
     PayLater,
     Wallet,
-    BankRedirect,
+    Klarna,
```

Comment:
`Klarna` and `Paypal` are not PaymentMethod. Those are `payment_method_type`

Reply:
Removed klarna and paypal